### ISSUE TITLE
TTSN and TNI don't update when manually adding flights from previous days

### ISSUE THREAD

**Issue by markmacpherson** [cadets-ca/ets#35]
When adding flights from previous days, the Total Time Since New (TTSN) and the Time to Next Inspection (TNI) are not updated to indicate the new times.

**Comment by kirvanp** [comment]
Knowing how this should work is a dilemma. It seems obvious at first that if I add a flight last week, the TTSN for that day and every day since should be incremented by the duration of that flight. But what happens if the user has already manually adjusted the TTSN of the more recent days to be correct? Then incrementing would make them wrong. Let's take a real example:

I fly a cross country in the towplane to gliding site X. I don't have an iPad that day, so I just write the times in the paper Journey log. The next day, site X staff adjust the TTSN for that aircraft to match the journey logs and start flying. After a few weeks, someone says, "shouldn't that cross country be in the iPad?" so they enter it in. Now in this case all the timesheets they've made in the weeks from when they manually adjust the TTSN to when they enter the missing flight are correct and shouldn't be tampered with. 

We could solve this problem by not letting you manually adjust the TTSN, but that would make things less flexible and potentially make other errors harder to fix.

**Comment by markmacpherson** [comment]
Yes, something we need to explore further for sure. We should aim to keep as much flexibility as possible. Maybe even just popping up a reminder that they need to double check the TTSN and TNI after they add flights manually?

**Comment by kirvanp** [comment]
Yeah. There is a feature that allows you to set the initial TTSN of one entry to the final TTSN of the prior one so you can manually go through and adjust them if that’s the right thing to do. In addition, mismatches of final and initial TTSN are highlighted in red in the journey log entries list.
